doc/API.md

```diff
@@ -57,7 +159,7 @@ struct {
 'target' is the hash that needs to be met by a block for the block to be valid.
 The target is inversely proportional to the difficulty.
 
-#### /consensus/block [GET]
+#### /consensus/blocks/:height [GET]
```

Comment:
I think it makes sense to use `<>` in the docs, though it does make the path/nickname stuff more confusing